.evergreen/config.yml

```diff
@@ -117,8 +139,14 @@ pre:
            REMOTE_PATH="${project}/${build_variant}/${branch_name}/${tag_name}/libmongocrypt.tar.gz"
         fi
         unset tag_name
+        PROJECT_DIRECTORY="$(pwd)"
         echo "libmongocrypt_s3_path: $REMOTE_PATH"
-        echo "libmongocrypt_s3_path: $REMOTE_PATH" > expansion.yml
+        echo "project_directory: $PROJECT_DIRECTORY"
```

Comment:
Can you explain what's going on here?

Reply:
Yes, adding the current dir (project dir) to the expansion yaml file.  I believe its processed later via the call to `eval "${inject_env_from_evergreen}"`.  Its done differently in the java driver but not that differently.